community/kernel/src/main/java/org/neo4j/kernel/NeoStoreDataSource.java

```diff
@@ -694,6 +689,50 @@ private NeoStoreKernelModule buildKernel( LogFiles logFiles, TransactionAppender
         return new NeoStoreKernelModule( transactionCommitProcess, kernel, kernelTransactions, fileListing );
     }
 
+    private AtomicReference<CpuClock> setupCpuClockAtomicReference()
+    {
+        AtomicReference<CpuClock> cpuClock = new AtomicReference<>( CpuClock.NOT_AVAILABLE );
+        if ( config.get( GraphDatabaseSettings.track_query_cpu_time ) )
+        {
+            cpuClock.set( CpuClock.CPU_CLOCK );
```

Comment:
I would consider calling
```java
cpuClockUpdater( null, config.get( GraphDatabaseSettings.track_query_cpu_time ) );
```
to avoid the double `boolean` to `CpuClock` conversion.